api/tests/core/offerers/test_models.py

```diff
@@ -325,3 +325,23 @@ def test_dms_adage_status_when_multiple_dms_application(self):
         )
 
         assert venue.dms_adage_status == latest.state
+
+
+class OffererDmsAdageStatusesTest:
+    def test_dms_adage_statuses_when_no_dms_application(self):
+        offerer = factories.OffererFactory()
+        factories.VenueFactory(managingOfferer=offerer)
+
+        assert offerer.dms_adage_statuses == set()
+
+    def test_dms_adage_status_when_multiple_dms_application(self):
+        offerer = factories.OffererFactory()
+        venue_accepted = factories.VenueFactory(managingOfferer=offerer)
+        educational_factories.CollectiveDmsApplicationFactory(venue=venue_accepted, state="accepte")
+
+        assert offerer.dms_adage_statuses == set(["accepte"])
+
+        venue_rejected = factories.VenueFactory(managingOfferer=offerer)
+        educational_factories.CollectiveDmsApplicationFactory(venue=venue_rejected, state="refuse")
+
+        assert offerer.dms_adage_statuses == set(["accepte", "refuse"])
```

Comment:
```suggestion
        assert offerer.dms_adage_statuses == {"accepte", "refuse"}
```
c'est plus simple d'écrire directement un `set`.